zulip_bots/zulip_bots/lib.py

```diff
@@ -159,6 +161,48 @@ def state(self, default):
         yield new_state
         self.set_state(new_state)
 
+def setup_default_commands(bot_details, message_handler):
+    def def_about():
+        if bot_details['description'] == "":
+            return "**{}**".format(bot_details['name'])
+        return "**{}**: {}".format(bot_details['name'], bot_details['description'])
+
+    def def_help():
+        return ("\n".join("**{}** - {}".format(k, v[1])
+                          for k, v in defaults.items() if k) + "\n" +
+                "\n".join("**{}** - {}".format(k, v)
+                          for k, v in bot_details['commands'].items() if k))
```

Comment:
Let's use more verbose variable names than k/v here.  I think we want "command_name" and "command_description".  

I think some of the other new functions in this file could use the same sort of work.

As a general rule, one should basically always be verbose in variable names.  Brevity doesn't provide much value, and the extra clarity is huge.

Also, put the newline on its own line so it is more visible.